testsrc/main/mondrian/rolap/NonEmptyTest.java

```diff
@@ -6133,8 +6133,6 @@ public void testMon2202SeveralFilteredHierarchiesPlusMeasureFilter() {
     }
 
     public void testMon2202AnalyzerCompoundMeasureFilterPlusTopCount() {
-        propSaver.set(
-            propSaver.properties.AlertNativeEvaluationUnsupported, "ERROR");
         assertQueryReturns(
```

Comment:
@YuryBY 
The ERROR setting in this test is intentional.  We want to verify that the query below actually does use native evaluation.
You may need to increase the maxConstraints size of this test now that we have the check.